e2e/common/cluster.go

```diff
@@ -28,9 +34,13 @@ import (
 
 const defaultClusterName = "e2e-test"
 
+var NamespacesUsed []string
```

Comment:
Couldn't this be a field on the `Cluster` type, passed in as a parameter through `WithCluster`?

Reply:
yes, I've changed the way it was done pretty much, see my last commit